src/services/redirectToDestination.js

```diff
@@ -24,7 +25,17 @@ export function redirectToDestination(purposeParams, userID, redirectID){
             break;
         
         default:
-            throw new Error('Destination not supported') 
+            //if destination is invalid, then send an error log into SQS.
+            var purpose = 'RedirectError'
```

Comment:
Will you edit the code on the lambda to handle this purpose?

Reply:
I thought for now, we'll just let the message stay in the queue? The lambda will filter only if `purpose == 'attendance`

Reply:
Needs to be `redirctError` no?

You can remove the `throw`. This will just throw an error in the console log. Not useful to us

Reply:
Messages in the queue get auto deleted in some 15 days if I remember correctly, so ideally we should put them somewhere

Reply:
@pritamps I can't see a throw? 
@deepansh96 would it be a good idea to use sentry for this?